utils/moreutils/patches/001_disable-manuals.patch

```diff
@@ -0,0 +1,11 @@
+--- a/Makefile
```

Comment:
@nikil Thanks for the explanation.
@diizzyy At least provide proof for your claim. Some find and grep on both core and packages didn't show many packages where man-pages are disabled like this.

In general, I don't think it's worth the effort, unless it's required to fix breakage or avoid dependency bloat, so in this case it's fine.